cdk/resources/create_config_lambda/config.yaml

```diff
@@ -0,0 +1,115 @@
+---
+
+url: https://{issuer}
+
+tornado:
+  debug: false
+  port: 8081
+  xsrf: true
+  xsrf_cookie_kwargs:
+    samesite: strict
+
+auth:
+  require_jwt: true
+  set_auth_cookie: true
+  get_user_by_oidc: true
+
+oidc_secrets:
+  client_id: {oidc_client_id}
+  secret: {oidc_client_secret}
+  client_scope:
+    - email
+    - openid
+jwt_secret: {jwt_secret}
+
+get_user_by_oidc_settings:
+  metadata_url: {oidc_metadata_url}
+  jwt_groups_key: 'cognito:groups'
+  grant_type: authorization_code
+  id_token_response_key: id_token
+  access_token_response_key: access_token
+  access_token_audience: null
+
+redis:
+  host:
+    global: {redis_host}
+
+celery:
+  broker:
+    global: redis://{redis_host}:6379/2
+  active_region: {aws_region}
+
+ses:
+  region: {aws_region}
+  consoleme:
+    name: ConsoleMe
+    sender: consoleme_admin@{issuer}
+  support_reference: >-
+    Please contact us at consoleme_admin@{issuer} if you have any
+    questions or concerns.
+  arn: {ses_identity_arn}
+  support_chat_url: {support_chat_url}
+  documentation_page: https://hawkins.gitbook.io/consoleme/
+
+cloud_credential_authorization_mapping:
+  role_tags:
+    enabled: true
+    authorized_groups_tags:
+      - consoleme-authorized
+    authorized_groups_cli_only_tags:
+      - consoleme-owner-dl
+      - consoleme-authorized-cli-only
+  dynamic_config:
+    enabled: true
+  internal_plugin:
+    enabled: false
+
+application_admin: {application_admin}
+
+groups:
+  can_admin:
+    - consoleme_admins
+  can_edit_config:
+    - consoleme_admins
+  can_admin_policies:
+    - consoleme_admins
+
+aws:
+  account_number: {account_number}
+  region: {aws_region}
+  issuer: {issuer}
+  session_duration: 900
+
+get_eligible_roles:
+  from_config: true
+  from_prefix: false
+
+headers:
+  group_access:
+    enabled: true
+  policies:
+    enabled: true
+  config:
+    enabled: true
+
+enable_self_service: true
+
+user_role_creator:
+  default_trust_policy:
+    Version: '2012-10-17'
+    Statement:
+      - Effect: Allow
+        Principal:
+          AWS:
+            - arn:aws:iam::{account_number}:root
+        Action: sts:AssumeRole
+
+challenge_url:
+  enabled: true
+
+{spoke_accounts_objects_list_yaml}
```

Comment:
Seems like yaml linter doesn't like this, even though this whole templated string is formatted and generated dynamically.

Reply:
Since this is auto-generated, let's [add an `exclude` pattern](https://pre-commit.com/#pre-commit-configyaml---hooks) to the [first block in `.pre-commit-config.yaml`](https://github.com/Netflix/consoleme/blob/9948633db7c00ab417035f2c2bc747e71c039506/.pre-commit-config.yaml#L3-L9).

Reply:
I added the exclude to both `prettier` and `check-yaml` becaue main `exclude` parameter doesn't accept list of exclusions so I preferred to avoid regex hell.

Reply:
That sounds like the right call.